pom.xml

```diff
@@ -7,10 +7,48 @@
     <groupId>com.srutkowski.training</groupId>
     <artifactId>unittest</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <name>unittest</name>
 
     <properties>
         <maven.compiler.target>11</maven.compiler.target>
         <maven.compiler.source>11</maven.compiler.source>
     </properties>
 
+    <dependencies>
+       <dependency>
+        <groupId>org.junit.jupiter</groupId>
+       <artifactId>junit-jupiter-engine</artifactId>
+       <version>5.8.2</version>
+       <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.mockito</groupId>
+       <artifactId>mockito-core</artifactId>
+       <version>4.2.0</version>
```

Comment:
Thats very secure and wise to provide the concrete version of the used library. Good Job!